app/_redirects

```diff
@@ -2,6 +2,9 @@
 # Newer, more relevant redirects should appear at the top of the file.
 # More general/sweeping redirects should be at the bottom.
 
+#3.3
+/gateway/latest/upgrade/#upgrade-considerations-and-breaking-changes    /gateway/latest/production/breaking-changes/33x
```

Comment:
This redirect will expire when we move to 3.4

You can do 
/gateway/3.3.x/upgrade/#upgrade-considerations-and-breaking-changes /gateway/latest/production/breaking-changes/33x


Though it makes me wonder if we want one of the breaking changes docs urls to be: 

`/gateway/latest/production/breaking-changes/`

and use an origin url for it?

Reply:
@Guaris How would I create an origin URL? I've never done that before.

Reply:
It would work like this:

```
- text: Upgrade and Migration
  items:
    - text: Upgrade Kong Gateway 3.x.x
      url: /upgrade/
      src: /gateway/upgrade-v2
    - text: Migrate from OSS to Enterprise
      url: /migrate-ce-to-ke/
    - text: Breaking Changes
      url: /production/breaking-changes/
      src: /gateway/production/breaking-changes/33x
```
I guess we'd just need to update that for every release.

Though to combine this with the comment about the location of this content, I think it's more like this:
https://github.com/Kong/docs.konghq.com/pull/5833#discussion_r1276840488

And the entry for the current release gets the origin URL:
```
items:
  - text: Kong Gateway 3.3.x
    url: /breaking-changes/
    src: /gateway/breaking-changes/33x
```

Reply:
Coming back to put it all together, I _think_ this is what could work:

```
- title: Introduction
    icon: /assets/images/icons/documentation/icn-flag.svg
    items:
      ...
      - text: Release Notes
        url: /gateway/changelog
        absolute_url: true
      - text: Breaking Changes
        items:
          - text: Kong Gateway 3.3.x
            url: /breaking-changes/
            src: /gateway/breaking-changes/33x
          - text: Kong Gateway 3.2.x
            url: /breaking-changes/32x/
          - text: Kong Gateway 3.1.x
            url: /breaking-changes/31x/
          - text: Kong Gateway 3.0.x
            url: /breaking-changes/30x/
          - text: Kong Gateway 2.8.x or earlier
            url: /breaking-changes/28x/
```

Upgrade section:

```
- text: Upgrade and Migration
  items:
    - text: Upgrade Kong Gateway 3.x.x
      url: /upgrade/
      src: /gateway/upgrade-v2
    - text: Migrate from OSS to Enterprise
      url: /migrate-ce-to-ke/
    - text: Breaking Changes
      url: /production/breaking-changes/
      generate: false
```

redirects:

```
/gateway/latest/upgrade/#upgrade-considerations-and-breaking-changes    /gateway/latest/breaking-changes/
/gateway/latest/production/breaking-changes/     /gateway/latest/breaking-changes/
/gateway/:version/production/breaking-changes/   /gateway/:version/breaking-changes/
```